packages/config-loader/src/sources/FileConfigSource.ts

```diff
@@ -48,6 +49,11 @@ export interface FileConfigSourceOptions {
    * A substitution function to use instead of the default environment substitution.
    */
   substitutionFunc?: SubstitutionFunc;
+
+  /**
+   * A content parsing function to transform string content to configuration values.
+   */
+  parsingFunc?: ParsingFunc;
```

Comment:
Appreciate that we're trying to make this the same as `substitutionFunc`, but thinking that we shoud just call this `parser` instead to be honest, and a similar rename for the type.

Reply:
Updated.